nano/rpc_test/rpc.cpp

```diff
@@ -1710,36 +1710,64 @@ TEST (rpc, keepalive)
 TEST (rpc, peers)
 {
 	nano::test::system system;
-	auto node = add_ipc_enabled_node (system);
+	// Add node2 first to avoid peers with ephemeral ports
```

Comment:
How does changing the order avoid ephemeral ports?

Reply:
The way tests setup nodes is that second node connects to the first one. Connection is initiated from ephemeral port to a specific listening port. Here we can only predict the listening port, so node needs to connect to node2.

Reply:
If there is a better way I'll be happy to change it. I wanted to get it working without removing asserts.